Makefile

```diff
@@ -12,7 +12,8 @@ endif
 
 # GO_BUILD_ARGS should be set when running 'go build' or 'go install'.
 REPO = $(shell go list -m)
-VERSION = $(shell git describe --dirty --tags --always)
+#TODO (anrastog): set version to repo build/tag after v1 plugin is available in master.
+VERSION = master
```

Comment:
Hardcoded the image version to master as that is the only tag that exists for quay.io/joelanford/helm-operator right now.